sdk/storage/Azure.Storage.Blobs/src/PartitionedDownloader.cs

```diff
@@ -55,21 +55,21 @@ internal class PartitionedDownloader
             }
 
             // Set _rangeSize
-            if (transferOptions.MaximumTransferSize.HasValue
```

Comment:
I was actually okay with `Size` for these if you liked that better than `LongLength`.  I thought it was a really good alternative to `Length` that still made sense to users.  I only wanted to explicitly fix the `Long` suffixed properties.

Reply:
Okay I'll set that back.